meta-rdk-broadband/recipes-ccsp/unified-wifi-mesh-personality/unified-wifi-mesh-personality-common.bb

```diff
@@ -0,0 +1,23 @@
+inherit systemd
+DESCRIPTION = "Common runtime dependencies for unified-wifi-mesh for both controller and extender"
+LICENSE = "CLOSED"
```

Comment:
LICENSE cannot be CLOSED in an open repo (almost always).    Apache-2.0 should be used please.